package.json

```diff
@@ -100,7 +100,7 @@
     "check:test": "mocha --recursive test/functional test/unit",
     "check:ts": "tsc -v && tsc --noEmit",
     "check:atlas": "mocha --config \"test/manual/mocharc.json\" test/manual/atlas_connectivity.test.js",
-    "check:adl": "mocha test/manual/data_lake.test.js",
+    "check:adl": "mocha --exit test/manual/data_lake.test.js",
```

Comment:
Maybe this is a better place to ask this question. I think adding `--exit` has great potential for hiding bugs, did you file a ticket about the issue this is meant to address so you remember to remove `--exit` in the future?

Reply:
I was hoping the problem would go away on it's own before I merged these PRs. I'm running a patch build to check now, if it's still timing out I'll file a ticket.